src/main/java/com/dtolabs/rundeck/plugin/overthere/OTWinRMNodeExecutor.java

```diff
@@ -604,7 +631,11 @@ public String getWinrmLocale() {
         public String getWinrmTimeout() {
             return resolveProperty(WINRM_TIMEOUT, null, getNode(), getFrameworkProject(), getFramework());
         }
-        
```

Comment:
(this difference is odd; double check the line endings of the file and the ones you are inserting. Isn't the file using \r\n, and you inserting only \n, or the inverse?)

Reply:
It is a cosmetics change: spaces on empty line

Reply:
(likewise: no cosmetics change in a purposeful change; please remove)